### ISSUE TITLE
Remove "No Actions detected" error

### ISSUE THREAD

**Issue by jhnns** [mheap/pin-github-action#28]
Hi 👋
first of all: thank you for this useful CLI helper :)

I wanted to ask if you could remove the "No Action detected" error. My use case is this: I have a `prepare` job that runs before the actual job and checks some preconditions:

```yml
jobs:
  prepare:
    runs-on: ubuntu-latest
    if: "!contains(github.event.head_commit.message, '[skip ci]') && ${{ github.event.workflow_run.conclusion == 'success' }}"
    steps:
      - run: echo "${{ github.event.head_commit.message }}"
```

Now `pin-github-action` throws an error because the `prepare` job doesn't use any action. Personally, I think `pin-github-action` should never throw an error if the workflow doesn't use an action. This makes it suitable for usage more similar to a code formatter. E.g. in my case I'm planning to execute `pin-github-action` as part of a [lint-staged](https://github.com/okonet/lint-staged) workflow.

What do you think?

**Comment by mheap** [comment]
I've prototyped this in #29 - would that be suitable for your use case?

**Comment by jhnns** [comment]
Yes, that's perfect 🙏

**Comment by mheap** [comment]
Released as `v1.4.0` 🥳
